WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostPreviewFragment.java

```diff
@@ -32,7 +37,7 @@ public static EditPostPreviewFragment newInstance(SiteModel site, PostModel post
         EditPostPreviewFragment fragment = new EditPostPreviewFragment();
         Bundle bundle = new Bundle();
         bundle.putSerializable(WordPress.SITE, site);
-        bundle.putSerializable(EditPostActivity.EXTRA_POST, post);
+        bundle.putInt(EditPostActivity.EXTRA_POST_LOCAL_ID, post.getId());
```

Comment:
> Since `PostModel` objects can be quite large, in order to avoid issues like `TransactionTooLargeException` it's better to pass the id of the post. However, we still want to keep passing the `SiteModel` to avoid confusion around local & remote ids.

@maxme Would a comment like the above one work for what you had in mind?